GrailDiary/GrailDiary/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,280 @@
+                                                            <nil key="textColor"/>
+                                                            <nil key="highlightedColor"/>
+                                                        </label>
+                                                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Country" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="fad-Xc-QK0">
+                                                            <rect key="frame" x="0.0" y="25" width="61.5" height="20.5"/>
+                                                            <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                                            <nil key="textColor"/>
+                                                            <nil key="highlightedColor"/>
+                                                        </label>
+                                                    </subviews>
+                                                </stackView>
+                                            </subviews>
+                                            <constraints>
+                                                <constraint firstItem="TRB-TC-Hr4" firstAttribute="centerY" secondItem="NIP-io-QSO" secondAttribute="centerY" id="0rh-c1-NGM"/>
+                                                <constraint firstItem="t1f-yV-ctG" firstAttribute="leading" secondItem="NIP-io-QSO" secondAttribute="leadingMargin" id="CQT-9l-lE6"/>
+                                                <constraint firstItem="TRB-TC-Hr4" firstAttribute="leading" relation="greaterThanOrEqual" secondItem="t1f-yV-ctG" secondAttribute="trailing" constant="8" id="GRp-xm-Gm9"/>
+                                                <constraint firstItem="TRB-TC-Hr4" firstAttribute="trailing" secondItem="NIP-io-QSO" secondAttribute="trailingMargin" id="jMQ-QX-wCb"/>
+                                                <constraint firstItem="t1f-yV-ctG" firstAttribute="centerY" secondItem="NIP-io-QSO" secondAttribute="centerY" id="tLF-TC-N05"/>
+                                            </constraints>
+                                        </tableViewCellContentView>
+                                        <accessibility key="accessibilityConfiguration" identifier="POICell"/>
+                                        <connections>
+                                            <outlet property="cluesCountLabel" destination="TRB-TC-Hr4" id="lNg-Rf-x2e"/>
+                                            <outlet property="countryLabel" destination="fad-Xc-QK0" id="dOR-Ko-CFS"/>
+                                            <outlet property="locationLabel" destination="fTU-JR-Mpo" id="La9-xR-QRD"/>
```

Comment:
![Screen Shot 2020-05-31 at 10 02 15 PM](https://user-images.githubusercontent.com/23512386/83369505-9511af80-a38a-11ea-8ee3-0c915a57690a.png)
It seems like your constraints are broken here
<img width="320" alt="Screen Shot 2020-05-31 at 10 04 13 PM" src="https://user-images.githubusercontent.com/23512386/83369538-af4b8d80-a38a-11ea-9c67-816e46076414.png">